src/main/java/emu/grasscutter/Config.java

```diff
@@ -74,7 +74,7 @@ public static class GameServerOptions {
 		public int InventoryLimitMaterial = 2000;
 		public int InventoryLimitFurniture = 2000;
 		public int InventoryLimitAll = 30000;
-		public int MaxAvatarsInTeam = 4;
+		public int MaxAvatarsInTeam = 9;
```

Comment:
Probably shouldn't be included

Reply:
![image](https://user-images.githubusercontent.com/73888354/166962700-6d2ce336-5c39-4891-8bbf-2d584c5a8210.png)

Maybe we shoud not limit MaxAvatarsInTeam on server. Client has some limit like 5. If you force add avatar into your team, the maxium is 5. When your team has 5, force join can not has response.